### PR TITLE
Hello Microverse project pull request

### PR DESCRIPTION
**Pull request by Santosh-Konappanavar**
Microverse Hello Project Pull Request 1

In this project, the following was done:

Created HTML and Style sheet files.
Incorporated Microverse HTML and CSS linters for code validation
-Added gitignore file to prevent the node/modules and test.md files from being commit to the git repository

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <h1>Hello Microverse</h1>
+</body>
```
Comment (roshan-bajgain) [inline index.html:12]: - [ ] [Optional] For better practice please use the class and style the class not the `h1` tag.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Santosh-Konappanavar, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
- [ ] Please use the more descriptive pull request please tell the reviewer what changes you made to the project like Added linter files, and created index.html file and style the heading etc .
### Pull request summary


- [ ] Please fix the MIT license in the README file as it is broken.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Santosh-Konappanavar) [reply index.html:12]: yeah sure

**Comment by Santosh-Konappanavar** [comment]
@roshan-bajgain could you please check the PR now

**Comment by Santosh-Konappanavar** [comment]
@roshan-bajgain I am not understood the MIT licence broken could you please explain it .

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @Santosh-Konappanavar  👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉 

## Highlights
✔️professional README👍 
✔️professional PR👍 
✔️No linter errors👍

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**